app/register/patient/patientForm.tsx

```
@@ -69,19 +101,27 @@ export default function PatientRegistrationForm() {
     try {
       const response = await fetch('/api/patient/register', {
         method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
         body: JSON.stringify({
           baseaddress: address,
           ...values,
+          height: parseFloat(values.height.toString()),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)